### ISSUE TITLE
Conditional Mean Spectrum (CMS)

### ISSUE THREAD

**Issue by hickeyj2** [usgs/earthquake-hazard-tool#265]
Hi. Apologies if this is the wrong place to post this. Just wondering if there are any plans to introduce the capability to calculate the CMS or exact CMS like the old (2008) deaggregation tool used to be able to do (or am I missing something?)? Thanks.

**Comment by pmpowers-usgs** [comment]
Yes. The first step is to get it (the exact CS) implemented in the underlying computational code (see [nshmp-haz #216](https://github.com/usgs/nshmp-haz/issues/216)) and then hook it into the web service. Users would initially be able to access it via the web service only. At some later date well add a plot that will accompany the 'total' and GMM-specific deaggregations.

This work is very high up the priority list so it should happen soon.

**Comment by hickeyj2** [comment]
That's great, thanks for the reply

**Comment by gareth-m** [comment]
Hello - Is there any update on this item??  Seems to have been open for a few years now

**Comment by emartinez-usgs** [comment]
@bclayton-usgs @pmpowers-usgs  ?

**Comment by pmpowers-usgs** [comment]
We are currently working on a significant refactor of the back-end codes for computing hazard and deaggregations, as well as this web application. As part of this we will be resuming support for conditional spectra. Thank you for the nudge.
